Makefile

```
@@ -24,13 +24,15 @@ ASSETS_DIR ?= assets
 default: all
 
 # Verify environment, and define PLUGIN_ID, PLUGIN_VERSION, HAS_SERVER and HAS_WEBAPP as needed.
-include build/setup.mk
+ifneq ($(MAKECMDGOALS),openapi-validate)
+	include build/setup.mk
 
 # The public/ directory contains the bridgeclient Go module for external consumption,
 # not HTTP assets. Override HAS_PUBLIC to prevent bundling these files.
 # TODO: Move bridgeclient to a top-level client/ directory for a cleaner import path.
 HAS_PUBLIC :=
 $(info Note: public/ directory contains Go modules, not HTTP assets - skipping bundle)
```

checkmake (0.3.2)

[warning] Target "$(info Note" should be declared PHONY.

(phonydeclared)